### ISSUE TITLE
CSV export just exports JSON with one row per image

### ISSUE THREAD

**Issue by jimjam-slam** [ryouchinsa/Rectlabel-support#193]
The CSV export appears to just have a column for image path and then an unquoted JSON array column for the labelled objects. It would be more helpful if this CSV was in a tidy format with one row per label.

For example, the [TFLite CSV spec](https://cloud.google.com/vision/automl/object-detection/docs/csv-format) takes the following columns with one row per label:

* Which dataset to assign the object to (training, validation, testing or let it decide);
* The image path;
* The object label;
* The bounding box, in standardised coordinates (0 to 1).

The export doesn't have to precisely follow this specification (especially the first column), but either (a) exporting with rows for each object rather than each image or (b) quoting the JSON array would make processing the export with standardised CSV tools a bit easier!

Thanks!

**Comment by ryouchinsa** [comment]
Thanks for writing the detailed feedback.

Currently we are porting our Caffe-based object detection code to Tensorflow and PyTorch.

Including the CSV export, we will update our exporting features for the latest version of both Tensorflow and PyTorch.

When we could implement, we will let you know.

**Comment by jimjam-slam** [comment]
No worries - thanks very much!

**Comment by ryouchinsa** [comment]
Hi,

We added Row by image and Row by label options to Export CSV file menu.
The new update version 32 was released.

![スクリーンショット 2022-03-28 21 44 55](https://user-images.githubusercontent.com/1954306/160401472-b552d868-4130-4fd2-9748-b0976c22a199.png)

When you select Row by label and check on the Convert to boxes checkbox, the format is the same as Tensorflow Object Detection CSV format.

```
filename,width,height,label,xmin,ymin,xmax,ymax
/Users/ryo/rcam/test_annotations/_raccoon/raccoon-1_jpg,650,417,raccoon,81,88,522,408
```

The details are here.
https://rectlabel.com/help#xml_to_csv

Let us know your opinion.

**Comment by jimjam-slam** [comment]
That looks great! I'll attempt to test when I can :)

**Comment by ryouchinsa** [comment]
Currently If you need our support to fix this problem, please let us know.
